common/solar_system_initializers/hfe_initializer.txt

```diff
@@ -69,100 +76,255 @@ hfe_initializer_egypt_main = {
 				species = event_target:hfe_egypt_species
+				authority = auth_imperial
 				ethos = {
 					ethic = ethic_fanatic_spiritualist
+					ethic = ethic_authoritarian
 				}
-				authority = auth_imperial
 				civics = {
 					civic = hfe_civic_egypt_genetic
 					civic = hfe_civic_egypt_rulers
 				}
+				origin = hfe_civic_egypt_rulers_origin
 				effect = {
+					save_global_event_target_as = hfe_egypt_country
+					set_graphical_culture = reptilian_01 # Learned from Ra'ar
+					set_city_graphical_culture = fallen_empire_03 # Light blue/red/yellow
 					set_country_flag = hfe_egypt
 					set_country_flag = hfe_assimilator_discovered
 					set_country_flag = hfe_marked_as_human
-					give_technology = {
-						tech = tech_titans
-					}
+#					hfe_effect_init_give_tech_t5 = yes
+					give_technology = { message = no tech = tech_titans }
+					give_technology = { message = no tech = tech_colossus }
+					give_technology = { message = no tech = tech_pk_cracker }
+#					give_technology = { message = no tech = tech_pk_neutron }
+					give_technology = { message = no tech = tech_psi_jump_drive_1 }
+					give_technology = { message = no tech = tech_psionic_shield }
+					give_technology = { message = no tech = tech_gene_tailoring }
+					give_technology = { message = no tech = tech_genome_mapping }
+					give_technology = { message = no tech = tech_vitality_boosters }
+					give_technology = { message = no tech = tech_epigenetic_triggers }
+					give_technology = { message = no tech = tech_cloning }
+					give_technology = { message = no tech = tech_gene_banks }
+					give_technology = { message = no tech = tech_gene_seed_purification }
+					give_technology = { message = no tech = tech_gene_expressions }
+					give_technology = { message = no tech = tech_selected_lineages }
+					give_technology = { message = no tech = tech_glandular_acclimation }
+					give_technology = { message = no tech = tech_genetic_resequencing }
+					# The tools to build and upload Thoth
+					give_technology = { message = no tech = tech_sapient_ai	}
+					give_technology = { message = no tech = tech_synthetic_workers }
+					give_technology = { message = no tech = tech_synthetic_leaders }
+					give_technology = { message = no tech = tech_droid_workers }
+					give_technology = { message = no tech = tech_robotic_workers }
+					give_technology = { message = no tech = tech_robomodding }
+					give_technology = { message = no tech = tech_robomodding_points_1 }
+					give_technology = { message = no tech = tech_robomodding_points_2 }
 					kill_leader = {
 						type = ruler
 						show_notification = no
 					}
 					create_leader = {
-						name = Amonhirkhopshef
+						name = "Amonhirkhopshef Mutatistep"
 						species = event_target:hfe_egypt_species
 						class = ruler
-						skill = 5
+						skill = 10
 						leader_age_min = 250
 						leader_age_max = 650
+#						immortal = yes # Had says it's automatically applied from the species
+						gender = male
```

Comment:
No gender is intended

Reply:
IF IT CREATES A FEMALE LEADER, IT KILLS IT AND CREATES A MALE LEADER ANYWAY.

Don't you ever talk to me about unnecessary code as long as this hypocrisy stands in the Mod.

Reply:
But then ruler will get number to his name, whats the point

Reply:
What number?

Reply:
In ruler name